app/react/settings/people/PersonCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link, BrowserRouter } from "react-router-dom";
+
+const PersonCard = data => (
+    <BrowserRouter>
+        <Link to={`/settings/usersEdit/${data.person.id}`}>
+            <div className="listing-item">
+                <div className="listing-item-primary">
+                    <div className="listing-item-image">
+                        <img
+                            className="avatar"
```

Comment:
The designs indicate we don't need the avatar anymore for the person in list view

Reply:
Right, but we are using the old designs so it made sense to have it